docs/settings/security-settings.asciidoc

```diff
@@ -337,24 +337,12 @@ For more details and a reference of audit events, refer to <<xpack-security-audi
 [cols="2*<"]
 |======
 | `xpack.security.audit.enabled` {ess-icon}
-| Set to `true` to enable audit logging for security events. *Default:* `false`
-|======
-
-[float]
-[[ecs-audit-logging-settings]]
-==== ECS audit logging settings
-
-To enable the <<xpack-security-ecs-audit-logging, ECS audit logger>>, specify where you want to write the audit events using `xpack.security.audit.appender`.
-
-[cols="2*<,*50"]
-|======
-| `xpack.security.audit.appender`
-| Optional. Specifies where audit logs should be written to and how they should be formatted.
+| Set to `true` and configure where the audit log should be written to using `xpack.security.audit.appender` to enable audit logging. *Default:* `false`
```

Comment:
I'm having a hard time parsing this sentence.  Can the part about enabling audit logging go first?

Set to `true` enable audit logging with `xpack.security.audit.appender` and configure where to write the audit log.